dist/stat_data.py

```diff
@@ -346,11 +279,86 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_pages_inuse', 'pages currently held in the cache', 'no_clear,no_scale'),
     CacheStat('cache_read_app_count', 'application threads page read from disk to cache count'),
     CacheStat('cache_read_app_time', 'application threads page read from disk to cache time (usecs)'),
-    CacheStat('cache_reentry_hs_eviction_milliseconds', 'total milliseconds spent inside reentrant history store evictions in a reconciliation', 'no_clear,no_scale,size'),
-    CacheStat('cache_timed_out_ops', 'operations timed out waiting for space in cache'),
     CacheStat('cache_write_app_count', 'application threads page write from cache to disk count'),
     CacheStat('cache_write_app_time', 'application threads page write from cache to disk time (usecs)'),
 
+    ##########################################
+    # Eviction statistics
+    ##########################################
+    EvictStat('eviction_active_workers', 'eviction worker thread active', 'no_clear'),
```

Comment:
It's hard to track down changes here. Just want to make sure that changes to stats only touch internal names, not external representation or flags.

Reply:
Ok I see it now from other comments.

Reply:
Yes, it touches only the internal names.